test/RaffleStore_test.js

```diff
@@ -93,18 +100,144 @@ describe("RaffleStore", function () {
     ).to.be.reverted
 
     await expect(
-      raffleStore.connect(rafflePlayer).enterRaffle(0, 1)
+      raffleStore.connect(rafflePlayer).enterRaffle(0, 1, {
+        value: ticketPrice.toString()
+      })
     ).to.be.satisfy
```

Comment:
Is the `.to.be.satisfy` needed? I don't really know this API but it reads weirdly haha

```suggestion
    )
```